workspaces/copilot/plugins/copilot-backend/README.md

```diff
@@ -15,10 +15,15 @@ To configure the plugin using the new backend system:
 
    const backend = createBackend();
 
-   backend.add(import('@backstage-community/plugin-copilot'));
+   backend.add(import('@backstage-community/plugin-copilot-backend'));
 
    backend.start();
    ```
+2. Add the package as a dependency in the backend workspace 
```

Comment:
Could this be `Step 1` so that it is consistent with the "Old system" instructions?

The wording could also be made consistent i.e. 

"Add the `@backstage-community/plugin-copilot-backend` package to your backend:"